docs/units/quantity.rst

```diff
@@ -254,17 +292,17 @@ Instead, only dimensionless values can be converted to plain Python scalars:
 Functions Accepting Quantities
 ------------------------------
 
-Validation of quantity arguments to functions can lead to many repetitons 
+Validation of quantity arguments to functions can lead to many repetitons
```

Comment:
Spelling error at end of line here (not your fault, but the removed space made it apparent).